paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -312,6 +311,16 @@ internal object ImageUtils {
     }
   }
 
+  fun scaleToThumbnailSize(image: BufferedImage): BufferedImage {
+    val scale = getThumbnailScale(image)
+    return scale(image, scale, scale)
+  }
```

Comment:
revert/inline?  there's only 1 usage of this.

Reply:
I'm a bit confused, inline the variable `scale` or the method `scaleToThumbnailSize `?

Reply:
`scaleToThumbnailSize`